templates/github/.github/workflows/kanban.yml.j2

```diff
@@ -0,0 +1,96 @@
+{% include 'header.j2' %}
+# Manage issues in a project board using https://github.com/leonsteinhaeuser/project-beta-automations
+
+---
+name: Kanban
+on:
+  pull_request_target:
+  issues:
+    types:
+      - labeled
+      - reopened
+      - assigned
+      - closed
+
+env:
+  free_to_take: Free to take
+  in_progress: In Progress
+  needs_review: Needs review
+
+jobs:
+  # only prio-list labeled items should be added to the board
+  add-to-project-board:
+    if: github.event_name == 'issues' && contains(github.event.issue.labels.*.name, 'prio-list') && contains(fromJson('["labeled", "reopened"]'), github.event.action)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add issue to Free-to-take list
+        uses: leonsteinhaeuser/project-beta-automations@v1.3.0
+        with:
+          gh_token: {{ "${{ secrets.RELEASE_TOKEN }}" }}
```

Comment:
with the current token I'm getting:
```
error validating token: missing required scope 'read:org'
```
probably the token needs more permissions 
FYI @RCMariko